src/renderers/shaders/ShaderChunk/lights_physical_pars_fragment.glsl

```diff
@@ -0,0 +1,50 @@
+struct PhysicalMaterial {
```

Comment:
Wonder if this file could be shared with lights_standard_pars_fragment with just a few changes if PHYSICAL is defined -- e.g. the struct could have extra members in that situation.